index.html

```diff
@@ -153,5 +114,6 @@
   <footer>
     We are the footer of the website.
   </footer>
+  <script src="js/app.js"></script>
```

Comment:
@omodara145 Please, lets move this back to the `<head>`, but you can make it a `defer` loaded script instead.